dawn/src/interface/atlas_interface.hpp

```diff
@@ -91,7 +91,14 @@ template <typename T>
 VerticalField<T> allocateFieldLike(atlasTag, const VerticalField<T>& other) {
   // leaky!
   auto field = new atlas::Field("copy", atlas::array::DataType::real64(), other.numElements());
-  return VerticalField<T>(atlas::array::make_view<double, 1>(*field));
+  return VerticalField<T>(atlas::array::make_view<T, 1>(*field));
+}
+
+inline VerticalField<::dawn::float_type> allocateField(atlasTag, size_t k_size) {
+  // leaky!
+  auto field = new atlas::Field("allocated", atlas::array::DataType::real64(),
```

Comment:
Does adding multiple atlas fields with the same name ("allocated") give problems? Is it tested?

Reply:
this does not give any problems, its just a tag to help with debugging